### ISSUE TITLE
Documentation website inaccessible due to error

### ISSUE THREAD

**Issue by hexbenjamin** [TomSchimansky/CustomTkinter#1521]
Hello!

Tried to go reference the docs just now, and was met with this error from the website. Thought you should know!

![image](https://user-images.githubusercontent.com/64101028/234946418-1dbe04cb-146e-43b7-83da-577bc76f3069.png)

**Comment by caiocv18** [comment]
I had the same error here using Google Chrome, but when I tested using Firefox, it worked. Anyway, I also leave here a contribution to let him know about this error.

**Comment by hexbenjamin** [comment]
Got it! @caiocv18

The docs site, since last night, has added an ads slider, and my copy of uBlock Origin was blocking the request to load it. The site will not run without that script loading successfully. So far, the ads are unintrusive and manageable. But yes, if you're using uBlock on chrome, maybe disable it for Tom's site. :)

**Comment by caiocv18** [comment]
Thanks a lot, @hexbenjamin!

**Comment by Binaris00** [comment]
Also, in Opera GX you need to disable this option to enter
![image_2023-04-29_110302201](https://user-images.githubusercontent.com/129613904/235309614-05e56d94-6d4e-4890-b2fe-bd37748bdce4.png)

**Comment by LeeviHalme** [comment]
Is there a reason why the documentation was moved off GitHub? I am currently using this package in a school project and would like the documentation to be available at all times.

In my opinion, GitHub is a more reliable place for storing the documentation, and I was greatly surprised by how detailed it was when I first looked into this repo :+1: 

This, and other browser-related issues others have had while reading the docs can be mitigated by just sticking to the GitHub wiki-page.
